vdiff/action.yml

```
@@ -357,14 +362,15 @@ runs:
         FORCE_COLOR: 3
         ORIGINAL_SHA: ${{ steps.run-info.outputs.original-sha }}
         SOURCE_BRANCH: ${{ steps.run-info.outputs.source-branch }}
-        TESTS_PASSED: ${{ steps.test-run.outputs.passed && steps.move-goldens.outputs.passed }}
+        TESTS_PASSED: ${{ steps.test-run.outputs.passed }}
+        FILES_MATCH: steps.move-goldens.outputs.passed }}
```

This is copy-pasted a few places:

```suggestion
        FILES_MATCH: ${{ steps.move-goldens.outputs.passed }}
```